docs/guide/topics/audit_logs.rst

```diff
@@ -0,0 +1,131 @@
+:orphan:
+
+.. currentmodule:: discord
+
+.. _guide_audit_logs:
+
+Audit Logs
+============
+
+Audit logs log various administrative actions taken in a guild. Examples include:
+
+- When a channel is created, updated, or deleted
+- When channel-specific permissions are created, updated, or deleted
+- When a member is kicked, pruned, banned, unbanned, updated, has their roles changed, or is moved or disconnected from a voice channel
+- When a bot is added to the guild
+- When a role is created, updated, or deleted
+- When an invite is created, updated, or deleted
+- When a webhook is created, updated, or deleted
+- When an emoji is created, updated, or deleted
+- When a message is deleted, bulk-deleted, pinned, or unpinned
+- When a guild integration is created, updated, or deleted
+- When a stage instance is created, updated, or deleted
+- When a sticker is created, updated, or deleted
+- When a scheduled event is created, updated, or deleted
+- When a thread is created, updated, or deleted
+    
+Depending on the action taken, an entry in the audit log may contain additional info about that specific action, including:
+
+
+
+.. warning:: 
+
+    There are a couple of limitations with audit logs that need to be understood:
+
+    1. There is no guarantee on audit logs arriving when expected (if they arrive at all)
+    2. No event is triggered when an audit log entry is created.
+    3. Audit logs are limited to 90 days.
+    4. No entry is created for message deletes if:
+
+      - it is a bot deleting a single message, or
+      - it is the message author deleting the message
+
+Each entry is an instance of :class:`AuditLogEntry`, which contains information about that particular entry, including:
```

Comment:
Similar to above, it feels a bit extraneous to just end up listing **everything** that falls under this umbrella.